app/src/main/java/com/hedvig/app/feature/offer/OfferItemsBuilder.kt

```diff
@@ -43,7 +43,8 @@ object OfferItemsBuilder {
                     .fragments
                     .monetaryAmountFragment
                     .toMonetaryAmount(),
-                incentiveDisplayValue = null,
+                incentiveDisplayValue = data.redeemedCampaigns.mapNotNull { it.fragments.incentiveFragment.displayValue },
+                hasCampaigns = data.redeemedCampaigns.isNotEmpty(),
```

Comment:
I think this will contain campaign for eg. Norways bundle? If that is the case we should use some other way of showing "Remove discount". I'll investigate.